lib/Dialect/FIRRTL/Import/FIRParserAsserts.cpp

```diff
@@ -203,9 +203,14 @@ namespace firrtl {
 /// Depending on the nature the verification operation, the `stop` may be
 /// optional. The Scala implementation simply removes all `stop`s that have the
 /// same condition as the printf.
-ParseResult foldWhenEncodedVerifOp(ImplicitLocOpBuilder &builder,
-                                   WhenOp whenStmt) {
-  auto *context = builder.getContext();
+ParseResult foldWhenEncodedVerifOp(PrintFOp printOp) {
```

Comment:
Not related to this patch, but while you're here, can you remove the enclosing namespaces and declare this function as:

`ParseResult circt::firrtl::foldWhenEncodedVerifOp(PrintFOp printOp) {`

?

Reply:
It looks like the prototype is in a .cpp file instead of a header, maybe that is why it was done this way.  You could move it to FIRAnnotations.h for example

Reply:
Move the declaration to header. Thanks!